src/plugin/fieldViews/NestedElementFieldView.ts

```diff
@@ -97,11 +97,14 @@ export class NestedElementFieldView extends ProseMirrorFieldView {
       getPos,
       offset,
       decorations,
-      outerView.state.plugins.filter((plugin) =>
-        plugin.spec.key === undefined
-          ? true
-          : [pluginKey, ...allowedPlugins].includes(plugin.spec.key)
-      ),
+      [
+        ...outerView.state.plugins.filter(
+          (plugin) =>
+            !plugin.spec.key ||
```

Comment:
I presume this is just a refactor to make the code easier to read?

But I'm confused about why you need to spread the array into a new array.

Reply:
good spot, this spead is left-over from when experimenting with a few explicitly added plugins, will address now